framework/widget/src/main/java/org/apache/ofbiz/widget/model/ModelFormFieldBuilder.java

```diff
@@ -729,7 +729,7 @@ public String getConditionGroup() {
      * Gets disabled.*
      * @return the disabled
      */
-    public boolean getDisabled() {
+    public FlexibleStringExpander getDisabled() {
```

Comment:
Rename to `getDisabledSpec` to indicate it is no longer returning the disabled status of a field.

Reply:
> A few changes requested around exposing disabled boolean state vs the specification of the disabled field attribute. These changes might have knock-on effects to callers which would also need clean up.

Changes made @danwatford, so now we're passing a boolean to HTML macros (most notable adjustment).